sp800-63-3/definitions.md

```diff
@@ -340,7 +340,7 @@ A set of policies, processes, server platforms, software and workstations used f
 The process of confirming the subscriber's continued presence and intent to be authenticated during an extended usage session.
 
 #### Registration
-The process through which an applicant applies to become a subscriber of a CSP and has their identity validated by the CSP.
+See [Enrollment](#enroll).
```

Comment:
Also need to update the definitions of _applicant_ and _biometrics_ that refer to registration, as well as a couple of biometrics references in -63B section 5.2.3